node/src/reactor.rs

```diff
@@ -713,16 +748,26 @@ where
                         break ReactorExit::ProcessShouldExit(ExitCode::Abort);
```

Comment:
Shouldn't we call `self.is_shutting_down.set();` just before this line too?

Reply:
Yes, good point. I missed that occurrence, overall I tried not to accidentally add new logic to the code outside of the intended changes, but this was an oversight.